.travis.yml

```diff
@@ -1,11 +1,12 @@
-sudo: false
+sudo: true
 cache: pip
 
 addons:
   apt:
     packages:
       - expect-dev  # provides unbuffer utility
       - python-lxml # because pip installation is slow
+      - ruby-sass
```

Comment:
I think this is not a valid package. You have to put it in install part. But let's wait the result.